### DISCUSSION TITLE
GPU compatible?

### DISCUSSION THREAD

**Discussion by christina-rice** [UvaCsl/HemoCell#108] [General discussion]
Has HemoCell been coded to run on GPUs? I was hoping to improve run time and have access to GPU cores.

**Comment by gzavo** [comment]
Hi @christina-rice,

We have started porting HemoCell to GPUs, but the performance tests were not fully convincing, and there were significant load-balancing difficulties between CPU and GPU parts, so the effort is on hold for now.
